bridge/e2e/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "outDir": "../out-tsc/e2e",
     "module": "commonjs",
-    "target": "es5",
+    "target": "es2018",
```

Comment:
Is this a requirement? AFAIK `es2018` is still not supported by all browsers.

Reply:
I think this is not relevant for browsers, as this is located in the e2e folder and used for e2e tests.